modules/oe_bootstrap_theme_paragraphs/src/EventSubscriber/IconOptionsSubscriber.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_bootstrap_theme_paragraphs\EventSubscriber;
+
+use Drupal\oe_paragraphs\Event\IconOptionsEvent;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Provides options for the icon field.
+ */
```

Comment:
I have questions about this mechanism, which I would like to see explained in the class doc or perhaps in a method doc.

1. Which icon field does this target, and how? From looking at other code I see this is targeting `field_oe_icon` which is defiend in oe_paragraphs. A `@see _oe_paragraphs_allowed_values_icons()` doc could help.
2. Which functionality from oe_paragraphs are we overriding here? Perhaps a `@see \Drupal\oe_paragraphs\EventSubscriber\OptionsSubscriber` doc can help.
3. Where does the list of icons come from? I assume we have an icons library somewhere in BCL? How do the machine names map to real icons? Is this a full list of icons available in bcl, or just a subset?

Reply:
Icons subset from BCL

Reply:
> 3. Icons subset from BCL
I see the icons in bootstrap-icons.svg.
I still don't see a doc in the code explaining this.. But ok.